resources/prosody-plugins/util.lib.lua

```diff
@@ -375,15 +375,15 @@ end
 -- Retrieves the focus from the room and cache it in the room object
 -- @param room The room name for which to find the occupant
 local function get_focus_occupant(room)
-    local focus_occupant = room._data.focus_occupant;
+    local focus_occupant_nick = room._data.focus_occupant_nick;
 
-    if focus_occupant then
-        return focus_occupant;
+    if focus_occupant_nick then
+        return room:get_occupant_by_nick(focus_occupant_nick);
     end
 
     for _, n_occupant in room:each_occupant() do
         if jid.node(n_occupant.jid) == 'focus' then
-            room._data.focus_occupant = n_occupant;
+            room._data.focus_occupant_nick = n_occupant.nick;
```

Comment:
Since we've hardcoded `focus` anyway, we could remove the cache 🤷

Reply:
I will look into that, yeah.